projects/css-custom-vars-viewer/scripts/parse-json.js

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const spectrumColours = [
+    'gray',
+    'blue',
+    'red',
+    'orange',
+    'yellow',
+    'chartreuse',
+    'celery',
+    'green',
+    'seafoam',
+    'cyan',
+    'indigo',
+    'purple',
+    'fuchsia',
+    'magenta',
+];
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const srcPath = path.resolve(__dirname, '../src/color-palette.json');
+const contents = fs.readFileSync(srcPath, { encoding: 'utf8' });
+const outputPath = path.resolve(__dirname, '../src/parsed-data.ts');
+
+const varsJSON = JSON.parse(contents);
+let colorArray = [];
+let exportString = '';
+
+spectrumColours.forEach((colour) => {
+    for (const [key, info] of Object.entries(varsJSON)) {
+        const colourSubstring = key.search(colour);
+
+        if (colourSubstring !== -1) {
+            const infoValues = info.value;
+            if (infoValues !== undefined) {
+                // eslint-disable-next-line no-console
+                console.log('invalid format!');
+            } else {
+                colorArray.push({
+                    customVar: `--spectrum-global-color-${key}`,
```

Comment:
```suggestion
                    customVar: `--spectrum-${key}`,
```
Should these be pointing to core color tokens?

Reply:
Yes. This is mainly just for an example project so I'm not too pressed about using the up-to-date tokens since they'll be deprecated anyway.